tests/foreman/ui_airgun/test_reporttemplates.py

```diff
@@ -407,12 +191,104 @@ def test_positive_e2e_crud(session, module_org, module_loc):
         assert pt['type']['snippet'] is False
         assert pt['locations']['resources']['assigned'][0] == module_loc.name
         assert pt['organizations']['resources']['assigned'][0] == module_org.name
+        # U
         session.reporttemplate.update(
             name,
             {'template.name': new_name, 'type.snippet': True}
         )
+        # check this by API once BZ1702676 gets fixed
+        # assert entities.ReportTemplate(name=new_name).search(
+        #    query={'search': 'name={0}'.format(new_name)})[0].read().snippet is True
         assert session.reporttemplate.search(new_name)[0]['Name'] == new_name
         updated_pt = session.reporttemplate.read(new_name)
         assert updated_pt['type']['snippet'] is True
+        # LOCK
+        session.reporttemplate.lock(new_name)
+        # check this by API once BZ1702676 gets fixed
+        # assert updated_pt.locked is True
+        assert session.reporttemplate.is_locked(new_name) is True
+        # UNLOCK
+        session.reporttemplate.unlock(new_name)
+        # check this by API once BZ1702676 gets fixed
+        # assert updated_pt.locked is False
+        assert session.reporttemplate.is_locked(new_name) is False
+        # CLONE
+        session.reporttemplate.clone(
+            new_name,
+            {
+                'template.name': clone_name,
+            }
+        )
+        crt = entities.ReportTemplate().search(
+                query={'search': 'name={0}'.format(clone_name)})[0].read()
+        assert crt.name == clone_name
+        assert crt.organization[0].read().name == module_org.name
+        assert crt.location[0].read().name == module_loc.name
+        assert crt.template == content
+        assert crt.default is True
+        # export
+        file_path = session.reporttemplate.export(new_name)
+        assert os.path.isfile(file_path)
+        with open(file_path) as dfile:
+            dfile_str = dfile.read()
+            assert content in dfile_str
```

Comment:
Maybe just `assert content in dfile.read()`? Or rename `dfile_str` to `dfile_content`.